source/1-SDLC-organization/lib/cdk-bootstrap-template.yml

```diff
@@ -0,0 +1,751 @@
+# This template is modified from original to add PermissionsBoundary for CFN Execution Role
```

Comment:
You only need to review two resources:
1. CloudFormationExecutionPermissionsBoundary
2. CICDPipelinePermissionsBoundary

The rest of the template is the same as the default one.